cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -274,7 +274,7 @@ func (command *Evaluate) Execute(args []string) (err error) {
 		totalScore = uint(len(languagesSelected))
 	}
 
-	assessmentsPerModel := assessments.Collapse()
+	assessmentsPerModel := assessments.CollapseByModel()
```

Comment:
Would rename commit to "refactor, New types for storing assessments per model and per language"

Reply:
I have no clue what commit you are currently talking about, Github doesn't link to the commit file but rather to all changes, we need to add the commit hash in such comments in the future for better tracing.